pkgs/tools/misc/c2patool/default.nix

```diff
@@ -0,0 +1,68 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, rustPlatform
+, libiconv
+, darwin
+, openssl
+, pkg-config
+, git
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "c2patool";
+  version = "0.7.0";
+
+  src = fetchFromGitHub {
+    owner = "contentauth";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-BmuNXIUHOmSdw+Fzk9CwlW9Piyh5iyddbqjqkzwcGnw=";
+  };
+
+  cargoHash = "sha256-xsZ+0sL8D3NURRYv967PaMATPxVyLBamDh0Ze5xjy4E=";
+
+  # use the non-vendored openssl
+  OPENSSL_NO_VENDOR = 1;
+
+  # work around https://github.com/NixOS/nixpkgs/issues/166205
+  NIX_LDFLAGS = lib.optionalString (stdenv.cc.isClang && stdenv.cc.libcxx != null) " -l${stdenv.cc.libcxx.cxxabi.libName}";
+
+  nativeBuildInputs = [
+    git
```

Comment:
Why do we need git? It probably can't find useful information anyway

Reply:
One of the subdependencies has a build script that pulls in git submodules using git.